modules/components/appfac/org.wso2.carbon.appfactory.core/src/main/java/org/wso2/carbon/appfactory/core/util/CommonUtil.java

```diff
@@ -67,21 +67,19 @@ public static String getApplicationType(String applicationId, String tenantDomai
 
         GenericArtifactImpl artifact = getApplicationArtifact(applicationId, tenantDomain);
 
-        if (artifact == null) {
-            String errorMsg = String.format("Unable to find application information for id : %s", applicationId);
-            log.error(errorMsg);
-            throw new AppFactoryException(errorMsg);
-        }
-
         try {
-            String applicationType = artifact.getAttribute("application_type");
-            return applicationType;
+	        if (artifact != null) {
```

Comment:
Since getApplicationArtifact method getting the artifact from Registry for an application id and tenantDomain, ApplicationManager.getApplicationType wouldn't be used to get the application type.